hack/jenkins/test-history/gen_history

```diff
@@ -19,10 +19,16 @@
 
 set -o errexit
 set -o nounset
+set -o xtrace
```

Comment:
Why? It only adds ~10 lines of output to this file, and makes it easier to debug failures.